templates/financial-services/index.html

```diff
@@ -92,67 +92,67 @@ <h2 class="p-muted-heading u-align--center">INNOVATING ON UBUNTU WITH CANONICAL<
     <div class="col-8">
       <h2>Drive business agility</h2>
       <h4>A better multi-cloud strategy</h4>
-      <p>You need two public clouds and an efficient private cloud. Ubuntu is optimised on AWS, Azure, Google, IBM and Oracle, with seamless portability. For on-premise IAAS, the <a href="/engage/cloud-economics">451 Research Cloud Price Index</a> shows Canonical delivers the best value.</p>
+      <p>Financial institutions need hybrid multi-cloud strategy - two public clouds and an efficient private cloud to increase business agility, manage risk, reduce total cost of ownership and keep up with the pace of digital transformation. Ubuntu is optimised on AWS, Azure, Google, IBM and Oracle cloud providing seamless portability. For on-premise Infrastructure-as-a-Service (IaaS), the <a href="/engage/cloud-economics">451 Research Cloud Price Index</a> shows Canonical delivers the best value.</p>
       <p>
         <a href="/public-cloud">Public cloud&nbsp;&rsaquo;</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="/openstack">Private cloud&nbsp;&rsaquo;</a>
       </p>
-      <div class="p-heading-icon">
-        <div class="p-heading-icon__header">
-          {{ image (
-            url="https://assets.ubuntu.com/v1/a6517959-451-research-vector-logo.svg",
-            alt="Research 451",
-            width="86",
-            height="32",
-            hi_def=True,
-            loading="lazy",
-            attrs={"style": "margin-right: 1rem"}
-            ) | safe
-          }}
-          <div class="p-heading-icon__title u-no-margin">
-            <a href="/engage/finserv-multi-cloud">Find out why 60% of financial services turn to multi-cloud</a>
-          </div>
-        </div>
-      </div>
-    </div>
-    <div class="col-4 u-align--center u-hide--small">
-      <ul class="p-inline-images u-no-margin">
-        <li class="p-inline-images__item">
+      <ul class="p-inline-images u-align--left u-hide--small">
+        <li class="p-inline-images__item u-no-margin--left">
           {{ image (
           url="https://assets.ubuntu.com/v1/c26cade5-1024px-BNP_Paribas.svg1_.png",
           alt="BNP Paribas",
-          width="167",
-          height="37",
+          width="184",
+          height="41",
           hi_def=True,
           loading="lazy"
           ) | safe
           }}
         </li>
-      </ul>
-      <ul class="p-inline-images u-no-margin">
-        <li class="p-inline-images__item">
+        <li class="p-inline-images__item" style="margin-top: 0.75rem;">
```

Comment:
This hack is not great. I think you should do it with `display: flex` and `align-items: baseline` or similar.
![image](https://user-images.githubusercontent.com/40214246/109831106-07bffc00-7c37-11eb-82a5-de83ccb893dd.png)